scripts/aws-deploy.sh

```diff
@@ -0,0 +1,288 @@
+
+# Osparc config
+$psed -i -e "s/WEBSERVER_LOGIN_REGISTRATION_CONFIRMATION_REQUIRED=.*/WEBSERVER_LOGIN_REGISTRATION_CONFIRMATION_REQUIRED=$WEBSERVER_LOGIN_REGISTRATION_CONFIRMATION_REQUIRED/" .env
+$psed -i -e "s/WEBSERVER_LOGIN_REGISTRATION_INVITATION_REQUIRED=.*/WEBSERVER_LOGIN_REGISTRATION_INVITATION_REQUIRED=$WEBSERVER_LOGIN_REGISTRATION_INVITATION_REQUIRED/" .env
+$psed -i -e "s/WEBSERVER_STUDIES_ACCESS_ENABLED=.*/WEBSERVER_STUDIES_ACCESS_ENABLED=$WEBSERVER_STUDIES_ACCESS_ENABLED/" .env
+
+# Rabbit
+$psed -i -e "s/RABBIT_PORT=.*/RABBIT_PORT=$RABBIT_PORT/" .env
+$psed -i -e "s/RABBIT_HOST=.*/RABBIT_HOST=$RABBIT_HOST/" .env
+$psed -i -e "s/RABBIT_LOG_CHANNEL=.*/RABBIT_LOG_CHANNEL=$RABBIT_LOG_CHANNEL/" .env
+$psed -i -e "s/RABBIT_PROGRESS_CHANNEL=.*/RABBIT_PROGRESS_CHANNEL=$RABBIT_PROGRESS_CHANNEL/" .env
+$psed -i -e "s/RABBIT_USER=.*/RABBIT_USER=$RABBIT_USER/" .env
+$psed -i -e "s/RABBIT_PASSWORD=.*/RRABBIT_PASSWORD=$RABBIT_PASSWORD/" .env
+
+# Reddis
+$psed -i -e "s/REDIS_HOST=.*/REDIS_HOST=$REDIS_HOST/" .env
+$psed -i -e "s/REDIS_PORT=.*/REDIS_PORT=$REDIS_PORT/" .env
+
+
+# docker-compose-simcore
+ori_compose_simcore=`cat docker-compose.deploy.yml`
+
+# Traefik does not use https
+$psed -i -e 's/traefik.http.routers.${PREFIX_STACK_NAME}_webserver.entrypoints=.*/traefik.http.routers.${PREFIX_STACK_NAME}_webserver.entrypoints=http/' docker-compose.deploy.yml
+$psed -i -e 's/traefik.http.routers.${PREFIX_STACK_NAME}_webserver.tls=.*/traefik.http.routers.${PREFIX_STACK_NAME}_webserver.tls=false/' docker-compose.deploy.yml
+
+# We don't use a auto-generated root certificate for storage
+$psed -i -e 's/\s\s\s\ssecrets:/    #secrets:/' docker-compose.deploy.yml
+$psed -i -e 's/\s\s\s\s\s\s- source: rootca.crt/      #- source: rootca.crt/' docker-compose.deploy.yml
+$psed -i -e "s~\s\s\s\s\s\s\s\starget: /usr/local/share/ca-certificates/osparc.crt~        #target: /usr/local/share/ca-certificates/osparc.crt~" docker-compose.deploy.yml
+$psed -i -e 's~\s\s\s\s\s\s- SSL_CERT_FILE=/usr/local/share/ca-certificates/osparc.crt~      #- SSL_CERT_FILE=/usr/local/share/ca-certificates/osparc.crt~' docker-compose.deploy.yml
+
+new_compose_simcore=`cat docker-compose.deploy.yml`
+new_env_simcore=`cat .env`
+if [ "$ori_env_simcore" = "$new_env_simcore" ] && [ "$ori_compose_simcore" = "$new_compose_simcore" ]; then
+    echo "Simcore service ready for deployment"
+else
+    echo "Some values weren't up to to date in Simcore service (.env and/or docker-compose.deploy). They have been updated, please push them to github and restart the script"
+    exit
+fi
+
+popd
+
+
+# -------------------------------- PORTAINER ------------------------------
+echo
+echo -e "\e[1;33mstarting portainer...\e[0m"
+pushd ${repo_basedir}/services/portainer
+$psed -i -e "s/MACHINE_FQDN=.*/MACHINE_FQDN=$MACHINE_FQDN/" .env
+$psed -i -e "s/MONITORING_DOMAIN=.*/MONITORING_DOMAIN=$MONITORING_DOMAIN/" .env
+make up-aws
+popd
+
```

Comment:
for later... I think we should switch to jinja2 or use envsubst instead of these fragile sed stuff..